photoshell/photo.py

```diff
@@ -17,17 +17,29 @@
 ])
 
 
+photo_cache = {}
+
+
 class Photo(_Photo):
 
     # TODO: move this off the class
```

Comment:
While this part of the code is being touched, maybe it makes sense to go ahead and do some of these TODO's?